ilc/package.json

```diff
@@ -5,11 +5,11 @@
   "main": "server/app.js",
   "scripts": {
     "test": "cross-env NODE_ENV=test mocha",
-    "test:watch": "cross-env NODE_ENV=test mocha --watch",
+    "test:watch": "npm run test -- --watch",
     "test:coverage": "cross-env NODE_ENV=test nyc mocha",
-    "test:client": "cross-env NODE_ENV=test karma start --single-run",
-    "test:client:watch": "cross-env NODE_ENV=test karma start --no-single-run",
-    "start": "node --max-http-header-size 30000 server/app.js",
+    "test:client": "npm run build:systemjs && cross-env NODE_ENV=test karma start",
+    "test:client:watch": "npm run test:client -- --no-single-run",
+    "start": "NODE_ENV=production node --max-http-header-size 30000 server/app.js",
```

Comment:
This is not ok as it prevents k8s to override NODE_ENV

Reply:
We have the issue with `config` package
This package tries to get development config instead of default when NODE_ENV is not provided even if don't provide NODE_ENV=development

Reply:
Yes, I know it, but nevertheless we can't keep it here